enterprise/server/remote_execution/vfs/vfs_unix.go

```diff
@@ -244,9 +299,26 @@ func (vfs *VFS) Unmount() error {
 		return nil
 	}
 
+	vfs.logStats()
+
 	return vfs.server.Unmount()
 }
 
+func (vfs *VFS) startOP(path string, op string) {
+	if !vfs.logFUSEPerFileStats {
+		return
+	}
+
+	vfs.mu.Lock()
+	defer vfs.mu.Unlock()
+	stats, ok := vfs.perFileStats[path]
+	if !ok {
+		stats = make(perOpStats)
+		vfs.perFileStats[path] = stats
+	}
+	stats[op]++
```

Comment:
should we do this accounting only if it's enabled so we avoid the allocations otherwise?

Reply:
same in startOp

Reply:
i don't mean this to feel nitpicky -- i think we suspect (based on profiles jim sent) that we're doing a lot of allocations during setup of each input root and then the GC is struggling a bit to delete everything after we're done with it.

Reply:
`logFUSEPerFileStats` is not enabled by default and we skip this code when it's off...

Reply:
🤦🏻‍♂️

startOp still applies but maybe the cost there is nbd?

Reply:
I have run profiles during testing and have not seen this come up at all.